x-pack/plugin/core/src/test/java/org/elasticsearch/xpack/cluster/routing/allocation/DataTierAllocationDeciderTests.java

```diff
@@ -74,7 +76,7 @@
     public void testClusterRequires() {
         ClusterState state = prepareState(service.reroute(ClusterState.EMPTY_STATE, "initial state"));
         RoutingAllocation allocation = new RoutingAllocation(allocationDeciders, state.getRoutingNodes(), state,
-            null, 0);
+            null, null,0);
```

Comment:
NIT: Missing space here

Reply:
Spaces addressed in b942b2db21b646f910664c935f11e3919d329945 (sorry)